graphql/bin/create_and_push_docker_image.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+## =============================================================================
+## NAME: create_and_push_docker_image.sh
+##
+##
+## DESCRIPTION
+## -----------------------------------------------------------------------------
+## Builds the GraphQL image and pushes it to NASA Github Packages.
+##
+## USAGE
+## -----------------------------------------------------------------------------
+## bin/create_and_push_docker_image.sh
+##
+## This must be called from the (root) graphql directory /graphql
+##
+## Args:
+##   $1 - The version number to create in Github.
+##   $2 - An access token with the `write:packages` permission on the NASA organization.
+## =============================================================================
+
+## Set this for Debugging only
+#set -ex
+
+## Make sure we are calling the script the correct way.
+BASEDIR=$(dirname $0)
+if [ "$BASEDIR" != "bin" ]; then
+  >&2 echo "ERROR: This script must be called from the root directory of the component [bin/create_and_push_docker_image.sh]."
+  exit 1
+fi
+
+
+source ../bin/common/check_returncode.sh
+
+version_number=$1
+github_token=$2
+
+echo $github_token | docker login ghcr.io -u nasa --password-stdin
+check_returncode $? "Error logging into Github."
+
+image_name=orca-graphql
```

Comment:
What if we change `image_name` value to ` ghcr.io/nasa/cumulus-orca/graphql:$version_number` We then can remove line 42 since it is already tagged with the correct name and then we can update 43 to be `docker push $image_name` Thoughts?

Reply:
Additional research/implementation card required.

Reply:
Changed